bool charge_write # bật sạc
bool write_out1  